pkg/utils/printer/printer.go

```diff
@@ -99,6 +101,32 @@ func (p *JSONPrinter) Print(v interface{}) error {
 	return nil
 }
 
+func (p *JSONPrinter) Infof(format string, a ...interface{}) {
+	flag := viper.GetBool(config.ArgVerbose)
+	var toPrint = ToPrint{}
+	if flag {
+		str := fmt.Sprintf("[INFOF] "+format, a...)
+		toPrint.Message = str
+		err := WriteJSON(&toPrint, p.Stdout)
```

Comment:
All verbose/debugging content should go to `stderr` to make it easier for the user to actually separate useful content that can be parsed in automation from debug messages.

Reply:
done